stable/insights-agent/values.yaml

```diff
@@ -1,3 +1,5 @@
+disableServiceMesh: true
```

Comment:
can we move this down? maybe in to the `cronjobs` section?

Reply:
I forgot about the cronjobs section :) yeah that should work.